pkg/riskanalysis/cmd.go

```
@@ -71,12 +77,33 @@ func (opt *Options) Run() error {
 		return errors.Wrap(err, "error marshalling results")
 	}
 
-	req, err := http.NewRequest("GET", opt.SippyURL, bytes.NewBuffer(inputBytes))
+	req, _ := http.NewRequest("GET", opt.SippyURL, bytes.NewBuffer(inputBytes))
```

Any reason to ignore an error here?